data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/clemuvvis_sweden.asset

```diff
@@ -7,11 +8,22 @@ local layer = {
   Settings = {
     Gamma = 1.14,
     Multiplier = 1.4
-  }
+  },
+  Description = [[The Clementine Ultraviolet/Visible (UVVIS) Version 2 mosaic is a 
+  grayscale data set representing the albedo (brightness of the lunar surface) as 
+  measured at the 750 nanometer (nm) wavelength by the UVVIS camera (Lee, et al., 2009). 
+  Resolution of this mosaic is 118 meters per pixel (m).]]
 }
 
-asset.onInitialize(function () 
-  openspace.globebrowsing.addLayer(globeIdentifier, "ColorLayers", layer)
-end)
+assetHelper.registerLayerAndExport(asset, layer, globeIdentifier, "ColorLayers")
 
-asset.export("layer", layer)
+
+asset.meta = {
+  Name = "Clem Uvvis [Sweden]",
+  Version = "1.1",
+  Description = [[Moon Clementine UVVIS Global Mosaic 118m v2 map of the Moon. This map 
+  is hosted on the OpenSpace server in Utah.]],
```

Comment:
This is the Sweden version of this asset, so the description should be updated. However, the other assets with a Sweden/Utah version does not have this as part of the description

Reply:
Ah, saw that you did it for basically every layer after this one, heh 😄